src/auto-evo/EvolutionaryTree.cs

```diff
@@ -245,29 +252,33 @@ public override void _Process(float delta)
     private void SetupTreeNode(Species species, EvolutionaryTreeNode? parent, int generation,
         bool isLastGeneration = false)
     {
-        var node = treeNodeScene.Instance<EvolutionaryTreeNode>();
+        var position = new Vector2(generation * GENERATION_SEPARATION, 0);
+
+        // Make sure the next line won't end in an exception.
```

Comment:
This comment could be a bit clearer.

Reply:
That's a leftover comment that I forgot to delete.